### DISCUSSION TITLE
academy bug

### DISCUSSION THREAD

**Discussion by Fillox112** [betfinio/public#110] [Bugs and Reports]
Regarding the newbies academy, the "swap pol for bet tokens" step. If there is a person who already has bets in staking from the past and currently has 0 bet tokens in his wallet, he cannot complete this lesson because it says he must have some bet tokens. 

It should look not only at the balance of the wallet, but also at the staking.

<img width="625" alt="Snímek obrazovky 2024-12-19 v 20 07 24" src="https://github.com/user-attachments/assets/db88c9e9-77a7-450b-91fb-21b83221770b" />

**Comment by bf-monster** [comment]
Hello. hmm. good suggestions. We will add change to backlog, thank you!

**Comment by bf-gabagool** [comment]
The issue has been fixed. Thanks for letting us know!
